inc/blocks/namespace.php

```diff
@@ -382,6 +431,23 @@ function get_block_type( $post ) : string {
 	return str_replace( 'altis/', '', $type );
 }
 
+/**
+ * Readable block name based on the XB name.
+ *
+ * @param string $block_name Name of the XB.
+ * @return string
+ */
+function get_block_name( $block_name ) {
```

Comment:
I'd make this more specific as to what it's doing, might be a bit confusing if you're passing `$block_name` to the function, so perhaps `get_block_type_label()`.
```suggestion
function get_block_type_label( string $block_name ) : string {
```

Note adding the type hints too, always good practice for a function like this one.

Would be good to update the function in the dashboard namespace too to use this one so we only need to update a single function in future.